script/Ciel/Day 3/03_01_CIEL03_3_2.txt

```diff
@@ -28,7 +28,7 @@ But my heart is screaming.
 -- Page 107, Offset 5104.
 -- 真実をはき出さなければ破裂すると謂わんばかりに、その言葉を、のど元まで―――
 It's as though it's going to explode if I don't get the truth out.
- Those words well up in my throat―――
+ Words well up in my throat―――
```

Comment:
Maybe `those` since I think it's referring specifically to the end of the sentence he starts a couple lines above 
> If this blood is real, then there's no denying it. With my own two hands, I――― (killed her)